### ISSUE TITLE
Outdated libraries/headers

### ISSUE THREAD

**Issue by BioBox** [pspdev/psplinkusb#15]
I have issues compiling this because the pspsh makefile uses the out of date `-lcurses` library, which my linker can't find even though I have all the development packages. Changing it to `-lncurses` fixes it. Is there any reason not to change this?

I also have problems with the libusb requirement. This software uses libusb 0.1, but my os only has the dev package for libusb 1.0 which is **not** backwards compatible.

**Comment by sharkwouter** [comment]
Try libusb-compat. That provides what pspsh needs.

**Comment by carstene1ns** [comment]
Porting to libusb-1.0 api is possible, just nobody working on it currently.
On my system (Arch) curses library is symlinked to ncurses, so I did not find this issue yet. However, just changing it in the Makefile should be sufficient here.

**Comment by BioBox** [comment]
@sharkwouter yeah you're right. Tried that before making this post, but I tend to forget to grab the `-devel` version.
Anyways it compiles for me now. I might update the libusb if I'm in the mood.
